cashews/backends/redis/backend.py

```diff
@@ -171,6 +171,14 @@ async def delete_match(self, pattern: str):
         if keys:
             return await self._client.unlink(keys[0], *keys[1:])
 
+    async def get_match(self, pattern: str):
+        keys = []
+        async for key in self.keys_match(pattern):
```

Comment:
I like this idea with 'get_match' , that's why I think it should be optimized for redis backend.
What I would like to suggest - use scan inside this method and make bunch operation to get values , with all that keys that one iteration of scan returning:
```
SCAN 0 MATCH {pattern}
MGET {result of last scan}
SCAN x MATCH {pattern}
MGET {result of last scan}
....
```

Reply:
By the way, should I add using `count` as a param? I think that this hardcoded value is not a good fit sometimes.
I propose rewriting this function something like this:

```python
async def keys_match(self, pattern: str, count=100):
    cursor = b"0"
    while cursor:
        cursor, keys = await self._client.scan(cursor, match=pattern, count=count)
        for key in keys:
            yield key
```

Reply:
> use scan inside this method and make bunch operation to get values

I've made a simple test and discovered that this approach works **_slower_**. Probably it depends on the amount of matched keys.
Mostrly perfomance depends on the `count` parameter.

Reply:
Sure , good idea